File: Coding Projects/Section 10/RPGProject/Priest.cpp
//
// Created by dbehl on 5/14/2022.
//

#include "Priest.h"

Priest::Priest(string name, race Race) : Player(name, Race, 100, 200) {
}

string Priest::attack() const {
    return "I will assault you with Holy Wrath!";
}

File: Coding Projects/My Projects/delhi/delhi/Priest.cs
using System;
namespace rpg
{
	public class Priest : Player
	{
		public Priest(string name, race Race) : base(name, Race, 100,200)
		{}
		override public string attack()
        {
			return "I will assault you with Holy Wrath!";

		}
	}
}

